codeexamples/SYCLBufferExampleExecutingCUDAKernels/StaticLibraryUse/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+
+## [main]
+if(NOT DEFINED ${CMAKE_BUILD_TYPE})
+	set(CMAKE_BUILD_TYPE "Debug")
+endif()
+
+# Almost all CMake files should start with this
+# You should always specify a range with the newest
+# and oldest tested versions of CMake. This will ensure
+# you pick up the best policies.
+cmake_minimum_required(VERSION 3.11...3.27)
+
+message(STATUS ">>SYCLBufferExecCUDAKernels build started")
+
+# set the project name
+project(SYCLBufferExecCUDAKernels
+        VERSION 1.0
+        DESCRIPTION "SYCL Buffer model program executing CUDA kernels"
+        LANGUAGES CXX)
+
+# If you set any CMAKE_ variables, that can go here.
+# (But usually don't do this, except maybe for C++ standard)
+## [start set variables]
+# Let's nicely support folders in IDEs
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+set(TARGET_NAME CUDAInteropSYCLSimple)
+set(STATIC_LIB_INCLUDE_DIR "${CMAKE_SOURCE_DIR}/StaticLibraryMakeCUDAKernel/src")
+set(STATIC_LIB_DIR "${CMAKE_SOURCE_DIR}/StaticLibraryMakeCUDAKernel/bin")
+set(CMAKE_CXX_COMPILER icpx)
+set(COMPILE_FLAGS "-fsycl")
+string(APPEND COMPILE_FLAGS " -fsycl-targets=nvptx64-nvidia-cuda")
+string(APPEND COMPILE_FLAGS " -Xsycl-target-backend=nvptx64-nvidia-cuda --cuda-gpu-arch=sm_75")
+string(APPEND COMPILE_FLAGS " -fdiagnostics-color=always")
+string(APPEND COMPILE_FLAGS " -I${STATIC_LIB_INCLUDE_DIR}")
+string(APPEND COMPILE_FLAGS " -I/usr/local/cuda/include")
```

Comment:
It has been written this way to make it easier to isolate then reinsert compile flags in case of problems.
Noted about the CMake target_compile_options(). The approach used was taken from an Intel DPC++ example.